src/ServicePulse.Host/app/js/views/archive/controller.js

```diff
@@ -149,15 +149,17 @@
 
 
         vm.retrySelected = function () {
-            serviceControlService.retryFailedMessages(vm.selectedIds);
-            vm.selectedIds = [];
+            serviceControlService.retryFailedMessages(vm.selectedIds)
+                .then(() => {
+                    vm.selectedIds = [];
```

Comment:
Do we have a loading spinner here? What does the screen show while waiting for `retryFailedMessages` to complete? I can't test this because of issues with the select row function.